README.adoc

```diff
@@ -105,37 +104,46 @@ In the sources folder, you create a resource file `src/main/resources/applicatio
 include::complete/src/main/resources/application.properties[]
 ----
 
-Here, `spring.jpa.hibernate.ddl-auto` can be `update`, `create`, `create-drop`, refer to the Hibernate documentation for details.
+Here, `spring.jpa.hibernate.ddl-auto` can be `none`, `update`, `create`, `create-drop`, refer to the Hibernate documentation for details.
+
+`none` This is the default for `MySQL`, no change to the database structure.
+`update` Hibernate changes the database according to the given Entity structures.
+`create` Creates the database every time, but don't drop it when close.
+`create-drop` Creates the database then drops it when the `SessionFactory` closes.
+
+We here begin with `create` because we don't have the database structure yet. After the first run, we could switch it to `update` or `none` according to program requirements. Use `update` when you want to make some change to the database structure.
+
+The default for `H2` and other embedded databases is `create-drop`, but for others like `MySQL` is `none`
+
+It is good security practice that after your database is in production state, you make this `none` AND REVOKE ALL PREVILEGES FROM THE USER CONNECTED TO THE DATABASE, THEN GIVE HIM ONLY CREATE, UPDATE, INSERT, DELETE.
```

Comment:
Spelling mistake: "privilege". Please don' use CAPITALS. Maybe if this is important we should make it part of the guide - show the process of doing it on the command line like you did with the database setup?